docs/site/Update-generator.md

```diff
@@ -0,0 +1,68 @@
+---
+lang: en
+title: 'Update generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Update-generator.html
+---
+
+{% include content/generator-create-app.html lang=page.lang %}
+
+### Synopsis
+
+The `lb4 update` command runs against a LoopBack 4 project and checks
+dependencies against the installed `@loopback/cli`. By default, it checks for
+exact match. Use `--semver` option to check compatibility using semver
+semantics.
+
+```sh
+lb4 update [options]
+```
+
+The key benefit of `lb4 update` over `npm update` is that `lb4 update` honors
+the collection of compatible LoopBack packages (and their dependencies) that are
+released with the current lb4 command from `@loopback/cli`.
+
+If the command is run without a LoopBack 4 project, it prints the version of the
+current CLI and versions of LoopBack packages that are released with the
+`@loopback/cli`. This is the same as `lb4 -v`.
+
+### Options
+
+`--semver` : _(Optional)_ Use semver semantics to check version compatibility
+for project dependencies of LoopBack modules
```

Comment:
I find this little bit puzzling. What does semver semantics mean in this case? What is the default behavior (when the option is not present), how is it different from semver? What criteria can our users apply to decide which mode to choose?

Reply:
By default, `lb4 update` checks exact match of versions between the project dependencies and the ones from the running cli template. Settting `--semver` relaxes it by semver checks, for example, `^1.0.0` won't be updated as it's compatible with `1.0.1`.